### ISSUE TITLE
This is not an issue, just a thank you

### ISSUE THREAD

**Issue by ecrevisseMiroir** [2e3s/awatcher#1]
I couldn't find your contacts on github to thank you, so I'm reaching out here.

You don't know how much you've helped me, I've struggled to make ActivityWatcher work on GNOME.

Thanks.

**Comment by 2e3s** [comment]
Thank you, I'm happy that it works on Gnome right away, because I didn't test it much on Gnome.

<details>
<summary>There's a way to know Git email on Github if it's not obfuscated</summary>

Take any commit URL on Github, e.g. https://github.com/2e3s/awatcher/commit/38a2f1f1dac43e567c0fc29be229a9787e9a254b and add `.patch` to that URL.

</details>

**Comment by pesader** [comment]
Yeah, I share the sentiment! I could never get Activity Watch working on my own, while awatcher **just worked**.

Thanks a lot!

**Comment by cameronj86** [comment]
Thanks a lot for the watcher.

As a new user, is it possible to answer the difference between this and the stock activity watcher for the average Linux user? 

I haven't experienced any issues w/ the original unlike others (running Debian - KDE & X11) but perhaps there's a tighter integration w/ Linux that I'm overlooking (ie: having to create my own rules for Dolphin, System Settings, etc)

**Comment by 2e3s** [comment]
@cameronj86 The primary "feature" is working on Wayland which has been the default environment for some years already. X11 is less performant, less secure and generally outdated. The stock activity watcher is X11-only, this one works with most of Linux distributions.
Otherwise, the only feature is the app/title rewrites for sensitive content or annoying unwarranted changes of window titles. Actually, I wanted to propose this into the server instead of client-side later.
Theoretically, compiled code on Rust may be slightly more performant, but it doesn't matter for such a small task. If the stock watcher doesn't adapt soon, I wanted to propose to rewrite it using the code from this watcher.

**Comment by 2e3s** [comment]
@cameronj86 The primary "feature" is working on Wayland which has been the default environment for some years already, while X11 getting gradually phased out. The stock activity watcher is X11-only, this one works with most of Linux distributions.
Otherwise, the only feature is the app/title rewrites for sensitive content or annoying unwarranted changes of window titles. Actually, I wanted to propose this into the server instead of client-side later.
If the stock watcher doesn't adapt soon, after some work I wanted to propose to rewrite it using the code from this watcher.

> ie: not having to create my own rules for Dolphin, System Settings, plasmashell, konsole etc

If I understand you correctly, you don't have to create any rules. It's only for sensitive content or eye candy/convenience.

**Comment by cameronj86** [comment]
Thanks for the clarification. I've been dragging my feet on resolving some Nvidia/Wayland issues, but w/ Wayland increasingly becoming the default, I'll tackle that and then double back to awatcher in a week or two. 

Yesterday was my aw 'install day', so things are slowly coming into focus. You were also correct in your initial post that I was confusing aw-watcher capabilities w/ category export files. I'll skim the internet fo aw-category-export files to streamline my categorization moving forward. Thanks again for breaking things down.

**Comment by alexander-newton** [comment]
+1 Thank you!
